modules/metering-use-mysql-or-postgresql-for-hive.adoc

```diff
@@ -23,8 +26,7 @@ spec:
         db:
           url: "jdbc:mysql://mysql.example.com:3306/hive_metastore"
           driver: "com.mysql.jdbc.Driver"
-          username: "REPLACEME"
-          password: "REPLACEME"
+          secretName: REPLACEME
```

Comment:
It does not need the quotes?

Reply:
It's not a string in this case, so it could go either way. I think this way makes it more apparent it's not a string.

Reply:
That field is a string - whether or not it needs quotes is probably a better question for the doc people.

Reply:
If it's a string then I think it needs to be in quotes, same as the other strings in the yaml file.